src/languages/en.ts

```diff
@@ -2086,6 +2088,45 @@ export default {
             updateCurrencyPrompt: 'It looks like your Workspace is currently set to a different currency than USD. Please click the button below to update your currency to USD now.',
             updateToUSD: 'Update to USD',
         },
+        changeOwner: {
+            changeOwnerPageTitle: 'Transfer owner',
+            addPaymentCardTitle: 'Enter your payment card to transfer ownership',
+            addPaymentCardButtonText: 'Accept terms & add payment card',
+            addPaymentCardReadAndAcceptTextPart1: 'Read and accept',
+            addPaymentCardReadAndAcceptTextPart2: 'policy to add your card',
+            addPaymentCardTerms: 'terms',
+            addPaymentCardPrivacy: 'privacy',
+            addPaymentCardAnd: '&',
+            addPaymentCardPciCompliant: 'PCI-DSS compliant',
+            addPaymentCardBankLevelEncrypt: 'Bank level encryption',
+            addPaymentCardRedundant: 'Redundant infrastructure',
+            addPaymentCardLearnMore: 'Learn more about our',
+            addPaymentCardSecurity: 'security',
+            amountOwedTitle: 'Outstanding balance',
+            amountOwedButtonText: 'OK',
+            amountOwedText: 'This account has an outstanding balance from a previous month.\n\nDo you want to clear balance and take over billing of this workspace?',
```

Comment:
I think the [copy](https://expensify.slack.com/archives/C21FRDWCV/p1710521136442169?thread_ts=1710435317.182139&cid=C21FRDWCV) included the amount
```suggestion
            amountOwedText: ({amount}) => `This account has an outstanding balance of ${amount} from a previous month.\n\nDo you want to clear balance and take over billing of this workspace?`,
```

Reply:
addressed